js/audio/Index.svelte

```diff
@@ -34,6 +34,7 @@
 	export let show_share_button = false;
 	export let editable = true;
 	export let waveform_options: WaveformOptions = {};
+	export let max_file_size: number | null = null;
```

Comment:
The `max_file_size` is passed in from the app, it's not in the component config. This is done so that the `upload` function in the client can know the max_file_size